packages/xsnap/test/test-xs-js.js

```diff
@@ -50,6 +51,61 @@ test('simple TextEncoder / TextDecoder are available', async t => {
   t.deepEqual(opts.messages, ['"Hello! 😊"', '"A\\u0000A"']);
 });
 
+test('Base64.encode', async t => {
+  const opts = options(io);
+  const vat = xsnap(opts);
+  t.teardown(() => vat.terminate());
+  await vat.evaluate(`
+    const encoder = new TextEncoder();
+    globalThis.handleCommand = inputBuffer => {
+      const outputString = Base64.encode(inputBuffer);
+      const outputUint8Array = encoder.encode(outputString);
+      globalThis.issueCommand(outputUint8Array.buffer);
+    };
+  `);
+  const inputUint8Array = new TextEncoder().encode('Hello, World! 😃🌍');
+  const expectedOutputString = encodeBase64(inputUint8Array);
+  await vat.issueCommand(inputUint8Array);
+  t.deepEqual(opts.messages, [expectedOutputString]);
+});
+
+test.failing('Base64.encode degenerate input case', async t => {
```

Comment:
Does this still fail? It looks like they tried to fix it:
https://github.com/Moddable-OpenSource/moddable/commit/a24ddbb6ca92e4e9412743d15275ad6c95923a03

Reply:
They succeeded!